src/libraries/pkg/test/testPackages.proj

```diff
@@ -3,19 +3,26 @@
 
   <ItemGroup>
     <PackagesToTest Condition="'$(PackagesToTest)' != ''" Include="$(PackagesToTest)" />
-    <!-- transport package not expected to be closure complete  -->
-    <ExcludePackages Include="Microsoft.AspNetCore.Internal.Transport" />
-    <ExcludePackages Include="$(ExcludePackages)" />
 
     <PackageReports Condition="'@(PackagesToTest)' == ''" Include="$(PackageReportDir)*.json" Exclude="@(ExcludePackages->'$(PackageReportDir)%(Identity).json')"/>
```

Comment:
Presumably you'll clean this up later when we eliminate the harvesting and no longer need the harvesting check below?